### ISSUE TITLE
[FavoriteMedia]: "Could not be compiled"

### ISSUE THREAD

**Issue by minnieoo** [Dastan21/BDAddons#257]
### Discord Build

Stable

### Discord Version

stable 303105 (1897d8a)

### Plugin Version

1.9.1

### Steps To Reproduce

1. open discord with better discord installed

### Describe the bug

The plugin no longer works. I am notified of a compiling issue upon starting discord:
```
TypeError: Cannot read properties of undefined (reading 'HTTP')
    at plugin (betterdiscord://plugins/FavoriteMedia.plugin.js:338:15)
    at eval (betterdiscord://plugins/FavoriteMedia.plugin.js:5713:13)
    at eval (betterdiscord://plugins/FavoriteMedia.plugin.js:5714:3)
    at N.requireAddon (betterdiscord/renderer.js:5:28601)
    at N.loadAddon (betterdiscord/renderer.js:5:6221)
    at N.loadAddon (betterdiscord/renderer.js:5:26269)
    at N.loadAllAddons (betterdiscord/renderer.js:5:9278)
    at N.initialize (betterdiscord/renderer.js:5:3093)
    at N.initialize (betterdiscord/renderer.js:5:25382)
    at Object.startup (betterdiscord/renderer.js:5:408589)
```

**Comment by Ryuktheshinigamix** [comment]
Same issue, sth in the recent update must have broken sth.

**Comment by ROMAiev** [comment]
Same issue

**Comment by NoahRoters** [comment]
Same issue, i think that that have something to do with the new const "s". This const have smth to do with media, but idk how to fix it. 
An update would be nice :)

**Comment by TheEmbracedOne** [comment]
Same issue here, waiting for a fix

**Comment by Thunder-McNeil-007** [comment]
Same issue as well. Hopefully there will be a fix soon since it was a really nice addon.

**Comment by bob-carter** [comment]
This at least looks like it's an issue with a plugin library being used based on what the error massage says. What version of ZeresPluginLibrary was this built with/for?

**Comment by WlSDXM** [comment]
is this ever going to be fixed (asking respectfully) ?

**Comment by Dastan21** [comment]
Working on a big update

**Comment by Thunder-McNeil-007** [comment]
Nice to hear Dastan!

Always liked the option to fave pictures as well so i dont have to post them again. Plus that i had pages in the gif section what made it easier to look for faved gifs.

**Comment by Rexxxyxyxy** [comment]
any estimate to when itll be done?

**Comment by Dastan21** [comment]
Fixed in v1.10.0.
